pages/organizations/College-based.html

```diff
@@ -0,0 +1,136 @@
+                <div class="org-name"><p id="samasa-name">College of Nursing</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/COM-logo.png">
+                <div class="org-name"><p id="usa-name">College of Medicine</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/COE-logo.png">
+                <div class="org-name"><p id="usa-name">College of Education</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/CAF-logo.png">
+                <div class="org-name"><p id="usa-name">College of Agriculture and Forestry</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/CAS-logo.png">
+                <div class="org-name"><p id="usa-name">College of Arts and Sciences</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/COC-logo.png">
+                <div class="org-name"><p id="usa-name">College of Communication</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/col-logo.png">
+                <div class="org-name"><p id="usa-name">College of Law</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/COD-logo.png">
+                <div class="org-name"><p id="usa-name">College of Dentistry</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+        </div>
+
+
+
+    <!--Report-->
+    <div class="report-container">
+        <button class="main__report">
+          <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 24 24" style="fill: rgba(0, 0, 0, 1)">
+            <path d="m14.303 6-3-2H6V2H4v20h2v-8h4.697l3 2H20V6z"></path>
+          </svg>
+          Report a problem with this page
```

Comment:
I noticed you included the status code style because of the report button. I have plans to modify and make it work. Will you be willing to take it? 

It's not as big as the pages you've built so you'll manage and be able to practice using semantic HTML tags.

Reply:
what modification nong?

Reply:
Basically, we'll transfer the report button styles to the `global.css`. It won't be in this branch but a separate one that will be created in the future (if you check Notion, I already added the task). 

It will need to be enhanced with JavaScript...